### ISSUE TITLE
Running limited set of rules does not work properly.

### ISSUE THREAD

**Issue by rwjblue** [ember-template-lint/ember-template-lint#851]
If you would like to run a limited set of rules against a file, you _should_ be able to do:

```
echo "module.exports = { rules: { 'no-shadowed-elements': true } };" > temp-templatelint-rc.js
./node_modules/.bin/ember-template-lint --config-path ./temp-templatelint-rc.js some/file/here.hbs
```

However, due to the fact that the linter sets up _all_ known plugins here we are still running rules that are not supposed to be enabled:

https://github.com/ember-template-lint/ember-template-lint/blob/e10fed889cb6a17fd4d3cffbaa17fbbbbd46e813/lib/index.js#L51-L54

That section above, should be tweaked to something like:

```js
    let rules = this.config.loadedRules;
    let astPlugins = [TransformDotComponentInvocation];

    let configuredRules = this.config.rules;
    for (let pluginName in configuredRules) {
```

This would ensure that only plugins with configured rules would run.

**Comment by lifeart** [comment]
on it

**Comment by rwjblue** [comment]
Fixed by #870
